x-pack/plugins/apm/public/components/shared/service_link.tsx

```diff
@@ -24,6 +31,8 @@ interface ServiceLinkProps {
   serviceName: string;
 }
 
+const serviceDroppedBucketName = '_other';
```

Comment:
Just a comment: We might want to extract this as a shared constant since we are also using it for the transactions table. But I can't think of a good place to put it. So ok to leave it for now

Reply:
IMHO you could at least export this variable and reuse it here x-pack/plugins/apm/public/components/app/service_inventory/index.tsx. 

I'd probably rename it to something like `OTHER_SERVICE_NAME`. WDYT @sqren ?

Reply:
@cauemarcondes Thats actually a very good point. Initially this was the only place where were using the bucket name and hence we did not export it. But recently i added the callout and now its definitely a use case to move this into a common exported constant. 

Can you guys recommend a place where we have other such variables exported or should i export it from here itself?

Reply:
For now, i have exported it out from this shared component and renamed the variable as @cauemarcondes suggested